ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -755,6 +755,7 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       FIPS_ENABLED: "true"
     test:
+    - ref: idp-openldap
```

Comment:
It is better to update job name so that we know the added config directly when debuggin from RP page.
```diff
- - as: aws-ipi-shared-vpc-phz-sts-fips-f14
+ - as: aws-ipi-shared-vpc-phz-sts-openldap-fips-f14
```

Reply:
Done